resources/assets/js/features/public-profiles/components/Passenger/PassengerProfileContainer.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Preloader from 'app/components/Preloader';
+import PassengerProfile from './PassengerProfile';
+import PassengerAdditionalInfo from './PassengerAdditionalInfo';
+import { getPassengerProfile } from '../../actions';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { getTranslate } from 'react-localize-redux';
+
+import "../../styles/public-profile.scss";
+
+class PassengerProfileContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            preloader: true,
+        }
+    }
+
+    componentDidMount() {
+        this.props.getPassengerProfile(this.props.id);
+        this.setState({
+            preloader: false
```

Comment:
preloader will be turned of immediately after component mount, but data request is async, so preloader will be hidden before data will be loaded